src/DependencyInjection/PusherFactory.php

```diff
@@ -18,6 +19,7 @@ class PusherFactory
      * @param array $config
      *
      * @return Pusher
+     * @throws PusherException
```

Comment:
Does it?

Reply:
Yeah there are a lot of cases when this exception will be thrown. It's hidden inside Pusher sdk lib =)